src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import connect from 'react-redux/es/connect/connect';
+import { addNotification } from '../../../Notifications';
+import '@patternfly/patternfly-next/utilities/Display/display.css';
+import '@patternfly/patternfly-next/utilities/Flex/flex.css';
+import { List } from 'react-content-loader';
+import ExpandableRulesCard from './ExpandableRulesCard';
+import { Card, CardBody, CardHeader } from '@patternfly/react-core';
+import { CommentSlashIcon, FrownOpenIcon } from '@patternfly/react-icons';
+import './advisor.scss';
+
+const SYSTEM_FETCH_URL = '/r/insights/platform/advisor/v1/system/';
+
+class InventoryRuleList extends Component {
+    state = {
+        expanded: true,
+        inventoryReportFetchStatus: 'pending',
+        inventoryReport: {}
+    }
+
+    componentDidMount() {
+        this.fetchEntityRules();
+    }
+
+    async fetchEntityRules() {
+        const { entity } = this.props;
+        try {
+            const inventoryReport = (await fetch(`${SYSTEM_FETCH_URL}${entity.id}/reports`)).data;
+            this.setState({
+                inventoryReport,
+                inventoryReportFetchStatus: 'fulfilled'
+            });
+        } catch (error) {
+            this.props.addNotification({
+                variant: 'danger',
+                dismissable: true,
+                title: '',
+                description: 'Inventory item rule fetch failed.'
+            });
+            this.setState({
+                inventoryReportFetchStatus: 'failed'
+            });
+        }
+    }
+
+    expandAll(expanded) {
+        this.setState({ expanded: !expanded });
+    }
+
+    buildRuleCards = () => {
+        const { inventoryReport, expanded } = this.state;
+        return (
+            <Fragment>
+                <div className="pf-u-display-flex pf-u-flex-direction-row-reverse">
```

Comment:
This doesn't appear to be showing up correctly, should be in upper right of the list

Reply:
You ought to `import '@red-hat-insights/insights-frontend-components/components/Advisor.css'` as well.

Reply:
Oh I see, mmhh the problem here is that we are currently using old version of pf-next in FE components. So you have to import '@patternfly/patternfly-next/utilities/Flex/flex.css` on your own so this text is on right

Reply:
👍 🤗